Assets/SEETests/TestLibGit2Sharp.cs

```diff
@@ -0,0 +1,52 @@
+﻿using UnityEngine;
+using NUnit.Framework;
+using LibGit2Sharp;
+using System.IO;
+using System.Text;
+using SEE.Utils.Paths;
+using DiffMatchPatch;
+
+namespace SEETests
+{
+    /// <summary>
+    /// Test cases (also used for exploration) for LibGit2Sharp.
```

Comment:
It's not truly a test, is it? You used it for exploration, didn't you? There are no Asserts here.

Reply:
You added it. So this can be deleted?

Reply:
Ah, I was the editor, was I? Right, now I remember it. Yes, it can be deleted.

Reply:
Alright.